osc-check_repo.py

```diff
@@ -51,6 +51,13 @@ def _check_repo_download(self, request):
 
     # XXX TODO - Rewrite the logic here, meanwhile set is to x86_64
     arch = 'x86_64'
+
+    if request.src_package in request.i686_only:
+        # check imported binaries from x86_64 could be safe, but you can not
+        # get rpmlint.log there, and before this step i586 repo should be
+        # are build succeeded. And we have to ignore x86_64 was excluded.
```

Comment:
Can you fix the comment? I do not get the `... and before this step i586 repo should be are build`